### DISCUSSION TITLE
1981-07-03-the-lena-and-leslie-show/

### DISCUSSION THREAD

**Discussion by giscus** [FanzOfLenaZavaroni/fanzoflenazavaroni.github.io#20] [General]
# 1981-07-03-the-lena-and-leslie-show/

Two nights of entertainment at Gloucester Hall in Fort Regent featuring the Lena and Leslie Show, with performances by Lena Zavaroni, Leslie Crowther and the Gary Lloyd Orchestra.

https://fanzoflenazavaroni.github.io/1981-07-03-the-lena-and-leslie-show/

**Comment by StephenMolloy1** [comment]
Just came across this on eBid;
https://www.ebid.net/ie/for-sale/lena-zavarone-with-leslie-crowther-show-gloucester-official-theatre-flyer-209940268.htm

**Reply by FanzOfLenaZavaroni** [reply]
Thanks for the link. <s>I will added a copy of the flyer to this post.</s>
